sigstore/_internal/oidc/oauth.py

```diff
@@ -48,16 +51,21 @@ def log_message(self, _format: str, *_args: Any) -> None:
         pass
 
     def do_GET(self) -> None:
+        logger.debug(f"GET: {self.path}")
```

Comment:
We should include headers and other details about the request here as well (query params?)

Reply:
Yeah, good idea. In the "normal" case there should be very little here (since we only handle the initial kickoff that redirects to sigstore + the response), but it'll help debug pathological cases.

Reply:
(The query params will already be present, since they're embedded in `self.path`)

Reply:
[96b93bd](https://github.com/sigstore/sigstore-python/pull/136/commits/96b93bda456078bc431da8927c972b2e383b5f85) adds the headers to the log.

(Tangent: we should think about how/if we want to warn users about potentially sensitive materials in their logs.)